src/main/java/seedu/address/ui/TaskCard.java

```diff
@@ -54,7 +54,11 @@ public TaskCard(Task task, int displayedIndex) {
         progress.getChildren().add(progressLabel);
 
         if (task.getDate() != null) {
-            dueDate.setText(LocalDate.now().until(task.getDate(), DAYS) + " Days");
+            String deadline = LocalDate.now().until(task.getDate(), DAYS) + " Day(s)";
+            dueDate.setText(deadline);
+            if (deadline.charAt(0) == '-') {
```

Comment:
Is checking the String or checking the integer/long returned by until function a better option?